presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -788,13 +788,104 @@ public void testSort()
         assertFunction("ARRAY_SORT(ARRAY [ARRAY [1], ARRAY [2]])",
                 new ArrayType(new ArrayType(INTEGER)),
                 ImmutableList.of(ImmutableList.of(1), ImmutableList.of(2)));
+        assertFunction(
```

Comment:
It looks like you duplicated your test code accidentally. This section repeats the `// with lambda function` section below.